java/src/main/java/gov/doc/ntia/sigmf/Global.java

```diff
@@ -293,27 +328,129 @@ public void setCalibrationDate(Date calibrationDate) {
     this.calibrationDate = calibrationDate;
   }
 
-  public String getDataFilePath() {
-    return dataFilePath;
+  public String getFilepath() {
+    return filePath;
   }
 
-  public void setDataFilePath(String dataFilePath) {
-    this.dataFilePath = dataFilePath;
-  }
-
-  public CoordinateSystem getCoordinateSystem() {
-    return coordinateSystem;
-  }
-
-  public void setCoordinateSystem(CoordinateSystem coordinateSystem) {
-    this.coordinateSystem = coordinateSystem;
+  public void setFilepath(String filePath) {
+    this.filePath = filePath;
   }
 
+  @Deprecated
   public Measurement getMeasurement() {
     return measurement;
   }
 
+  @Deprecated
   public void setMeasurement(Measurement measurement) {
     this.measurement = measurement;
   }
+
+  public String getClassification() {
+    return classification;
+  }
+
+  public void setClassification(String classification) {
+    this.classification = classification;
+  }
+
+  public Long getNumChannels() {
+    return numChannels;
+  }
+
+  public void setNumChannels(Long numChannels) {
+    this.numChannels = numChannels;
+  }
+
+  public String getDataset() {
+    return dataset;
+  }
+
+  public void setDataset(String dataset) {
+    this.dataset = dataset;
+  }
+
+  public Long getTrailingBytes() {
+    return trailingBytes;
+  }
+
+  public void setTrailingBytes(Long trailingBytes) {
+    this.trailingBytes = trailingBytes;
+  }
+
+  public Boolean getMetadataOnly() {
+    return metadataOnly;
+  }
+
+  public void setMetadataOnly(Boolean metadataOnly) {
+    this.metadataOnly = metadataOnly;
+  }
+
+  public GeoJsonPoint getGeolocation() {
+    return this.geolocation;
+  }
+
+  public void setGeolocation(GeoJsonPoint location) {
+    this.geolocation = location;
+  }
+
+  public String getCollection() {
+    return collection;
+  }
+
+  public void setCollection(String collection) {
+    this.collection = collection;
+  }
+
+  public void addDataProduct(DataProduct dataProduct) {
+    if (this.dataProducts == null) {
+      dataProducts = new DataProducts();
+    }
+    dataProducts.add(dataProduct);
+  }
+
+  public void removeDataProduct(DataProduct dataProduct) {
+    if (dataProducts != null) {
+      dataProducts.remove(dataProduct);
+    }
+  }
+
+  public DataProducts getDataProducts() {
+    return dataProducts;
+  }
+
+  public void addDigitalFilter(DigitalFilter df) {
```

Comment:
Should removeDigitalFilter() or setDigitalFilters() method be added? Currently no way to remove filters.

Reply:
Done